zk_prover/src/merkle_sum_tree/tree.rs

```diff
@@ -1,6 +1,7 @@
 use crate::merkle_sum_tree::{Entry, MerkleProof, Node};
 use halo2_proofs::halo2curves::bn256::Fr as Fp;
-use num_bigint::BigUint;
+
+use super::Asset;
```

Comment:
Can you use `use crate::merkle_sum_tree::Asset` rather than `super` ? Just for coherence with the rest of the library